run-bahmni.sh

```diff
@@ -157,14 +157,14 @@ function resetAndEraseALLVolumes {
 
 function restartService {
     # One can ONLY restart services in current profile (limitation of docker compose restart command). 
-    echo "Listing the running services from current profile (.env file) that can be restarted..."
+    echo "Listing the running services from current profile ($file file) that can be restarted..."
     docker compose ps
 
     echo "Enter the name of the SERVICE to restart:"
     read serviceName
     
     echo "Restarting SERVICE: $serviceName"
-    docker compose restart $serviceName
+    docker compose --env-file "$file" restart $serviceName
 
     if confirm "Do you want to see the service logs?"; then
         docker compose logs $serviceName -f
```

Comment:
Doesn't this command need --env-file option?

Reply:
Added file path option.